pkg/action/release_testing.go

```diff
@@ -58,5 +60,21 @@ func (r *ReleaseTesting) Run(name string) error {
 		return err
 	}
 
+	if r.Cleanup {
+		for _, h := range rel.Hooks {
+			for _, e := range h.Events {
+				if e == release.HookTest {
+					hookResource, err := r.cfg.KubeClient.Build(bytes.NewBufferString(h.Manifest))
```

Comment:
Non-blocking optimization nit: You could glue all the manifests together and then have a single `client.Delete` call. Totally non blocking though

Reply:
is there a good example on how to do that? I was afraid that injecting a newline between each manifest may break in certain cases but I'd love to lessen the number of calls to k8s if possible.

Reply:
We do it here: https://github.com/helm/helm/blob/63b751ded75c2a48045e896ae3e6418c370038a1/pkg/action/uninstall.go#L183-L192

Reply:
Technically speaking though, I think it does end up being the same amount of delete calls, but it does lessen the number of `Build` calls so that you are only deserializing once

Reply:
thank you for the example! I'll go do that now